src/Events/Controllers/EventsController.cs

```diff
@@ -99,8 +100,8 @@ public async Task<ActionResult<string>> Post([FromBody] CloudEvent cloudEvent)
         [ProducesResponseType(StatusCodes.Status400BadRequest)]
         [Produces("application/cloudevents+json")]
         public async Task<ActionResult<List<CloudEvent>>> Get(
+            [FromQuery, Required] string resource,
```

Comment:
What happens if the parameter is missing? Will the response be 400 Bad Request or will it be 404 Not Found?

Reply:
Good